tools/README.md

```diff
@@ -1,7 +1,19 @@
 # Discovery Tools
 
+This is a Go interface implementation to resolve image reference name.
```

Comment:
Instead of "image reference names", maybe:

    image names to potential [Merkle roots](../glossary.md#merkle-root).

or similar?